frontend/main.py

```diff
@@ -324,11 +342,11 @@ def send_email(
     )
 
 
-def save_correction_to_hadith_table(urn: int, corrected_val: str):
+def save_correction_to_hadith_table(urn: int, corrected_val: str, attribute: str):
     conn = pymysql.connect(**MYSQL_PROPS)
     cursor = conn.cursor()
-    query = "UPDATE bukhari_english SET hadithText = %(hadith_text)s WHERE englishURN = %(urn)s;"
-    cursor.execute(query, {"hadith_text": corrected_val, "urn": urn})
+    query = "UPDATE bukhari_english SET {attr} = '{corrected_val}' WHERE englishURN = {urn}".format(attr = attribute, corrected_val = corrected_val, urn = urn)
```

Comment:
we want to continue using paramters for values in the SQL query i.e. previous line 331 prevents SQL injection.

We should use a formatted string for injecting the attribute name which can't be paramterized.

Reply:
PR updated, help to review again 🙏🏽 jazakAllah khair brother